test/cli/template-delete/orchestrator-schedule/expected-stdout

```diff
@@ -0,0 +1,5 @@
+Plan for "delete template instance" operation:
+  × C main/extractor/ex-generic-v2/empty
+  × C main/other/keboola.orchestrator/orchestrator
+  × C main/other/keboola.orchestrator/orchestrator/schedules/scheduler
```

Comment:
E2E test ukazuje, že smaže i folder se schedule.

Reply:
Kromě tohoto to pěkně uvidíš když porovnáš soubory v `in` a `out`. (Tady ten changed files tree vlevo to zrovna ukazuje docela pěkně.) V `out` zůstane jen `om-config` který je obyčejný konfig co nepatří do šablony.